lib/lines.js

```diff
@@ -807,7 +790,29 @@ Lp.isEmpty = function() {
     return this.length < 2 && this.getLineLength(1) < 1;
 };
 
-Lp.join = function(elements) {
+function indentString(indent, { useTabs, tabWidth } = {}) {
+	var tabs = 0;
+	var spaces = indent;
+
+	if (useTabs) {
+		tabs = Math.floor(indent / tabWidth);
+		spaces -= tabs * tabWidth;
+	}
+
+	var result = "";
+
+	if (tabs > 0) {
+		result += new Array(tabs + 1).join("\t");
+	}
+
+	if (spaces > 0) {
+		result += new Array(spaces + 1).join(" ");
+	}
+
+	return result;
```

Comment:
Can you stick to four spaces per tab when adding new implementation code, please? I realize you're a fan of tabs (nothing wrong with that), but consistency is important here.

Reply:
> Can you stick to four spaces per tab when adding new implementation code, please? I realize you're a fan of tabs (nothing wrong with that), but consistency is important here.

I apologize in advance, but I really can't help emphasizing the enormous amount of irony in the statement "consistency is important here" in reference to indents in a PR that is targeting a design flaw that produces inconsistent indents.

Could issue #315 as well as this PR possibly be addressed since this project appears maintained? These issues have been around for six years now.